www/docs/usage/commandline.md

```diff
@@ -59,13 +59,32 @@ This can either be set to:
 For example:
 
 ```bash
-export TF_PRODUCT=opentofu
+export TF_PRODUCT="opentofu"
 tfswitch # Will install opentofu instead of terraform
 ```
 
+### `TF_LOG_LEVEL`
+
+`TF_LOG_LEVEL` environment variable can be set to override default log level.
+
+- Supported log levels:
+  - `INFO`:   includes `PANIC`, `FATAL`, `ERROR`
+  - `ERROR`:  includes `PANIC`, `FATAL`, `ERROR`, `WARN`, `INFO`
```

Comment:
Wait, error won't include INFO?
Are the keys just switched up between 71/72?

Reply:
Yeah, exactly =) Fixed via 625d9e1